web-frame/src/app-shared/core/components/progress-bar/progress-bar.component.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * @license
+ * Copyright (c) 2020 THREEANGLE SOFTWARE SOLUTIONS SRL
+ * Available under MIT license webFrame/LICENSE
+ */
+import { ChangeDetectionStrategy, Component, Input, OnInit } from '@angular/core';
+import { ITranslationService } from '~app-shared/translate';
+
+@Component({
+  selector: 'app-progress-bar',
+  styleUrls: ['./progress-bar.component.scss'],
+  template: '',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  host: {
+    '[attr.aria-valuetext]': 'translationService.translate(infoText)',
+    'attr.aria-busy': 'true',
+    'attr.aria-live': 'polite',
+    'attr.role': 'progressbar',
+  },
+})
+export class ProgressBarComponent implements OnInit {
+  @Input()
+  public infoText: string = 'GENERAL.LOADING_PAGE';
```

Comment:
This is a very simple component, the status bar just flickers in a 1.5s loop:

![sandbox-slow-pages-preview](https://user-images.githubusercontent.com/2186426/83036785-b188b180-a043-11ea-9eb2-2a270e8adbe8.gif)

When a UI library is added to the project, we can use this as a wrapping component for an improved progress bar like `<mat-progress-bar>`